Assets/MRTK/SDK/Features/UX/Scripts/InteractiveElement/BaseInteractiveElement.cs

```diff
@@ -270,11 +331,84 @@ public void ResetAllStates()
 
         #endregion
 
+        #region Button Setting Utilities
+
+        /// <summary>
+        /// Set the Clicked state and invoke the OnClicked event
```

Comment:
Mention that this sets and unsets the state right away, this is a bit of an unusual behavior worth mentioning.